test/generator/files/track_files_test.rb

```diff
@@ -24,15 +24,7 @@ def test_tests_version
 
       def test_example_solution
         subject = TestTrackFiles.new
-        expected_filename = FixturePaths.track + '/exercises/alpha/.meta/solutions/alpha.rb'
-        assert_equal expected_filename, subject.example_solution.filename
-      end
-
-      def test_legacy_example_solution
-        subject = TestTrackFiles.new
-        subject.exercise_name = 'beta'
-        expected_filename = FixturePaths.track + '/exercises/beta/example.rb'
-        assert_equal expected_filename, subject.example_solution.filename
+        assert_instance_of ExampleSolutionFile, subject.example_solution
```

Comment:
Right. I reverted it back to the pattern that these tests used to follow now that we no longer have multiple filename patterns.